plugins/maven/pom.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.mobidevelop.robovm</groupId>
+    <artifactId>robovm-parent</artifactId>
+    <version>2.2.1-SNAPSHOT</version>
+    <relativePath>../../</relativePath>
+  </parent>
+
+  <groupId>com.mobidevelop.robovm</groupId>
+  <artifactId>robovm-maven-all</artifactId>  
```

Comment:
This is the artifactId that has the parent robovm-maven-parent. The maven folder contains 3 other modules: junit, maven-resolver and robovm-maven-plugin. The robovm-maven-all is the parent of these 3.